LeaderboardBackend.Test/Features/Users/LoginTests.cs

```diff
@@ -0,0 +1,126 @@
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using System.Threading.Tasks;
+using LeaderboardBackend.Models.Entities;
+using LeaderboardBackend.Models.Requests;
+using LeaderboardBackend.Models.Validation;
+using LeaderboardBackend.Test.Fixtures;
+using LeaderboardBackend.Test.Lib;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+using NUnit.Framework;
+
+namespace LeaderboardBackend.Test.Features.Users;
+
+public class LoginTests : IntegrationTestsBase
+{
+    private const string LOGIN_URI = "/login";
+    private const string VALID_EMAIL = "valid@user.com";
+    private const string BANNED_EMAIL = "banned@user.com";
+    private const string VALID_PASSWORD = "P4ssword";
+
+    [OneTimeSetUp]
+    public void Init()
+    {
+        s_factory.ResetDatabase();
+
+        // Swap to creating users via the UserService instead of calling the DB, once
+        // it has the ability to change a user's roles.
+        using IServiceScope s = s_factory.Services.CreateScope();
+        ApplicationContext dbContext = s.ServiceProvider.GetRequiredService<ApplicationContext>();
+        dbContext.Users.AddRange(new[]
+        {
+            new User{
+                Email = VALID_EMAIL,
+                Password = BCrypt.Net.BCrypt.EnhancedHashPassword(VALID_PASSWORD),
+                Username = "Test User",
```

Comment:
Use valid data for testing. This is not a valid username. "Banned User" is not a valid username either. I suggest Test_User and Banned_User instead.